google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -1182,7 +1182,7 @@ private void doneCallback(Throwable finalStatus) {
                     <= maxRetryDuration.toMillis())) {
           this.conectionRetryCountWithoutCallback++;
           log.info(
-              "Retriable error "
+              "Connection is going to be reestablished with next request. Retriable error "
```

Comment:
The connection is temporarily closed with retriable error

Reply:
Done.